notifications_utils/template.py

```diff
@@ -137,6 +137,16 @@ def content_count(self):
         return len(self.content_with_placeholders_filled_in)
 
     def is_message_empty(self):
+
+        if not self.content:
```

Comment:
Optional: I don't see any added tests for this line of code, is there a small test that you can write?

Also, is it even possible to have this case?

Reply:
It’s tested by this line:
https://github.com/alphagov/notifications-utils/pull/760/files#diff-e5050dea996611c8bdf32cfe6e8214cdR1161

I don’t think it’s something that can really happen, but I didn’t want to remove an existing test case.